src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -504,20 +503,24 @@ def test_subscription_level_deployment(self):
             'params': os.path.join(curr_dir, 'subscription_level_parameters.json').replace('\\', '\\\\'),
             # params-uri below is the raw file url of the subscription_level_parameters.json above
             'params_uri': 'https://raw.githubusercontent.com/Azure/azure-cli/dev/src/azure-cli/azure/cli/command_modules/resource/tests/latest/subscription_level_parameters.json',
+            'ts': '/subscriptions/996a2f3f-ee01-4ffd-9765-d2c3fc98f30a/resourceGroups/giants-deep/providers/Microsoft.Resources/templateSpecs/BasicLinked01/versions/1.0',
```

Comment:
Could this resource be created dynamically instead of a fixed value?

Reply:
The resource cannot yet be dynamically created. We need to add the template-spec command module (CRUD operations) before the test cases can be updated.